Contract1.cdc

```diff
@@ -100,9 +142,15 @@ pub contract LavaFlow {
     pub resource Quest {
         pub let id: UInt64
         pub let name: String
+        // Max 3 requirements without double req on attribute
         // pub let requirements: {strength: UInt64, intelligence: UInt64, cunning: UInt64}
+        // ex requirements1: { strenght < 10 }
+        // ex requirements2: { strenght >= 5, intelligence < 8 }
+        // ex requirements3: { strenght <= 5, intelligence > 8, cunning = 6 }
+        // pub let requirements: [QuestRequirement]
+        //
         pub let description: String
-        pub let onFail: Bool
+        pub let onFail: Bool // REMOVE THIS BECAUSE IT'S TIED TO THE PLAYER
```

Comment:
why would this be better on the player? on fail, a quest can trigger something that happens to a player, another quest, or an item